website/docs/reference/resource-configs/entry-point.md

```diff
@@ -0,0 +1,33 @@
+---
+title: entry_point
+sidebar_label: "entry_point"
+id: entry-point
+---
+<VersionCallout version="1.11" /> 
+
+<File name='functions/<filename>.yml'>
+
+```yml
+functions:
+  - name: <function name>
+    config:
+      runtime_version: <string> # required for Python UDFs
+      entry_point: <string> # required for Python UDFs
+```
+
```

Comment:
can we give an example here? like a function with multiple sub functions?

Reply:
addressed https://github.com/dbt-labs/docs.getdbt.com/pull/8116/commits/d2f0a2bfe54af18e8fa7961585c723c8b7cd4e72